packages/block-editor/src/components/link-control/index.js

```diff
@@ -78,10 +106,11 @@ import LinkControlSearchInput from './search-input';
  * @property {boolean=}                   forceIsEditingLink     If passed as either `true` or `false`, controls the
  *                                                               internal editing state of the component to respective
  *                                                               show or not show the URL input field.
- * @property {WPLinkControlValue=}        value                  Current link value.
- * @property {WPLinkControlOnChangeProp=} onChange               Value change handler, called with the updated value if
- *                                                               the user selects a new link or updates settings.
- * @property {boolean=}                   showInitialSuggestions Whether to present initial suggestions immediately.
+ * @property {WPLinkControlValue=}                               value                  Current link value.
+ * @property {WPLinkControlOnChangeProp=}                        onChange               Value change handler, called with the updated value if
+ *                                                                                      the user selects a new link or updates settings.
+ * @property {boolean=}                                          showInitialSuggestions Whether to present initial suggestions immediately.
+ * @property {(title:string)=>WPLinkControlValue=}               createSuggestion       Handler to manage creation of link value from suggestion.
```

Comment:
This isn't the correct column alignment (the variable names should all be aligned with the previous ones). The type makes it difficult, yes, but we can extract the type if needed to a separate, shorter-named `@typedef` block.

Reply:
Ok.